lib/open_invoice/entities/invoice.rb

```diff
@@ -4,15 +4,15 @@ module OpenInvoice
   module Entities
     class Invoice < Base
       def index(opts = {})
-        request(:get, "/supplier/#{supplier_uuid}/invoices/page", opts)
+        request(:get, "/#{OpenInvoice::Configure::VERSION_1}/supplier/#{supplier_uuid}/invoices/page", opts)
```

Comment:
Why are these VERSION_1 whereas the base class is VERSION_2?

Reply:
Currently, some of Enverus's endpoints are on V1, and others are on V2 (currently only buyer & supplier).